ruby-challenges.rb

```diff
@@ -21,6 +35,16 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def no_vowel string
+  string.delete 'AaEeIiOoUu'
```

Comment:
I like how you group the vowels. I often see 'aeiouAEIOU'. How you group is more pleasing to the eyes!